.travis/validate_commit_message.py

```diff
@@ -18,7 +11,7 @@
 CHANGELOG_EXTS = [".feature", ".bugfix", ".doc", ".removal", ".misc"]
 
 sha = sys.argv[1]
-project = "pulp_rpm"
+project = "{{ redmine_project if redmine_project }}"
```

Comment:
Is it right?

Reply:
oh! it was from yesterday's change! @daviddavis, unfortunately, it was not pure python!

Reply:
the fix is here: https://github.com/pulp/plugin_template/pull/248